lib/handlers/router-entities/cache-dynamo.ts

```diff
@@ -0,0 +1,38 @@
+import { DynamoDB } from 'aws-sdk'
+
```

Comment:
I would create a new folder for this, maybe? I feel like the root of `router-entities` is getting too crowded